### PR TITLE
DOCSP-42282: java avs index type support

### PR DESCRIPTION
**Pull request by rustagir**
# Pull Request Info

[PR Reviewing Guidelines](https://github.com/mongodb/docs-java/blob/master/REVIEWING.md)

JIRA - <https://jira.mongodb.org/browse/DOCSP-42282>
Staging - 
[indexes](https://preview-mongodbrustagir.gatsbyjs.io/java/DOCSP-42282-searchindex-type/fundamentals/indexes/#create-a-search-index)
[whats new](https://preview-mongodbrustagir.gatsbyjs.io/java/DOCSP-42282-searchindex-type/whats-new/#std-label-java-version-5.2)

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Did you run a spell-check?
- [x] Did you run a grammar-check?
- [x] Are all the links working?
- [x] Are the [facets and meta keywords](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) accurate?

### CODE REVIEW

File: source/fundamentals/indexes.txt [hunk 1]
```diff
@@ -204,21 +204,25 @@ Multikey indexes behave differently from other indexes in terms of query coverag
...
 see the :manual:`Multikey Indexes page </core/index-multikey>` in the MongoDB manual.
 
+.. _java-search-indexes:
 .. _search-indexes:
 
 Atlas Search Indexes
 ~~~~~~~~~~~~~~~~~~~~
 
+You can programmatically manage your Atlas Search and Atlas Vector
+Search Indexes by using the {+driver-short+}.
```
Comment (jordan-smith721) [inline source/fundamentals/indexes.txt:214]: I don't think indexes should be capitalized here
```suggestion
Search indexes by using the {+driver-short+}.
```

**Review by jordan-smith721 — APPROVED** [review]
LGTM w/ one small fix

File: source/includes/fundamentals/code-snippets/SearchIndexMethods.java [hunk 2]
```diff
@@ -26,21 +26,36 @@ public static void main( String[] args ) {
             MongoCollection<Document> collection = database.getCollection(COLL_NAME);
 
             // start create-search-index
-            Document index = new Document("mappings",
+            Document searchIdx = new Document("mappings",
                     new Document("dynamic", true));
-            collection.createSearchIndex("myIndex", index);
+            collection.createSearchIndex("myIndex", searchIdx);
```
Comment (rozza) [inline source/includes/fundamentals/code-snippets/SearchIndexMethods.java:31]: 👍

**Review by rozza — APPROVED** [review]
LGTM!

**Pull request merged**